### PR TITLE
Automated source names

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR fixes the automated source names from GCN events.

### CODE REVIEW

File: services/gcn_service/gcn_service.py [hunk 1]
```diff
@@ -142,7 +142,8 @@ def poll_events(*args, **kwargs):
                         payload['notice_type']
                         == "gcn.notices.icecube.lvk_nu_track_search"
                     ):
-                        if payload.get("pval_bayesian", 1) > 0.05:
+                        # 3 sigma
+                        if payload.get("pval_bayesian", 1) > 0.003:
```
Comment (Theodlz) [inline services/gcn_service/gcn_service.py:146]: the exactly 0 is also there on purpose? Just checking.

Comment (Theodlz) [reply services/gcn_service/gcn_service.py:146]: went from .05 to .003

Comment (mcoughlin) [reply services/gcn_service/gcn_service.py:146]: @Theodlz yes 3 sigma :)

File: skyportal/handlers/api/gcn.py [hunk 2]
```diff
@@ -390,29 +390,25 @@ def post_skymap_from_notice(
                         'ra': ra,
                         'dec': dec,
                     }
-                elif any(
-                    [True if 'GRB' in tag.text.upper() else False for tag in tags]
-                ):
+                elif any([True if 'GRB' in tag.upper() else False for tag in tags]):
```
Comment (Theodlz) [inline skyportal/handlers/api/gcn.py:393]: why is `.text` not needed? Have the tags been converted to text somewhere above? Is that true for all notices (and not just JSON)?

File: skyportal/handlers/api/gcn.py [hunk 3]
```diff
@@ -427,7 +423,7 @@ def post_skymap_from_notice(
 
                 if source.get('id', None) is not None:
                     existing_source = session.scalars(
-                        Source.select(user).where(Source.id == source['id'])
+                        Source.select(user).where(Source.obj_id == source['id'])
```
Comment (Theodlz) [inline skyportal/handlers/api/gcn.py:426]: nice catch

File: skyportal/handlers/api/gcn.py (hunk 2 above)
Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py:393]: yeah these tags are all just the plain text.

Comment (mcoughlin) [reply skyportal/handlers/api/gcn.py:393]: This is only the case where its like gcnevent.tags.

Comment (Theodlz) [reply skyportal/handlers/api/gcn.py:393]: got it, yeah it was just confusing that .text all of a sudden had to go away but nothing else upstream of that seems to have changed.

**Review by Theodlz — APPROVED** [review]
LGTM 🚀 -> EDIT: lots of test failing, to fix before merging

**Review by Theodlz — CHANGES_REQUESTED** [review]
tests need to be looked at and fixed if necessary

**Comment by mcoughlin** [comment]
Merged in the other PR

**Pull request closed without merging**